### DISCUSSION TITLE
5.0.0-alpha.2

### DISCUSSION THREAD

**Discussion by awmleer** [ant-design/ant-design-mobile#3854] [Announcements]
- 特性
  - 新增了 umd 构建产物
  - 新增了 TextArea 组件
- 优化
  - 包结构调整，`lib` 目录更名为 `cjs` 🔔
  - 重构了 Steps 组件，横向模式下也支持了 icon 属性
- 修复
  - 修复了 Input 组件 placeholder 文字颜色过深的问题

---

- Feature
  - Add umd bundle
  - Add TextArea component
- Enhancement
  - Adjust package structure, `lib` folder get renamed to `cjs` 🔔
  - Refactor the Steps component, `icon` property is now supported in vertical mode
- Fix
  - Fix the placeholder text color of Input component

---

迁移建议

- 由于 `lib` 目录调整为了 `cjs` 目录，所以如果之前有引入过 `lib/index.css` 的话，需要替换为 `es/index.css` 或 `cjs/index.css`
- Steps 组件进行了重构，默认是只展示小圆点的，没有图标，如果需要展示图标的话需要手动传入

<hr /><em>This discussion was created from the release <a href='https://github.com/ant-design/ant-design-mobile/releases/tag/v5.0.0-alpha.2'>5.0.0-alpha.2</a>.</em>

**Comment by afc163** [comment]
👍 终于来了！
